src/index.js

```diff
@@ -967,8 +968,19 @@ function drawLastUpdated(lastUpdated) {
 
   display.textContent = relativeTime[LANG];
   display.setAttribute("title", lastUpdated);
-  display.setAttribute("data-en", relativeTime["en"]);
-  display.setAttribute("data-ja", relativeTime["ja"]);
+  i18next.addResource(
```

Comment:
I tried to fix the same thing by calling drawLastUpdated() in setLang(), but this is a better fix than mine.

Could you merge to the latest, and remove the call to drawLastUpdated() in setLang()? This isn't required any more.